contents/tutorials/android-analytics.md

```diff
@@ -0,0 +1,435 @@
+
+```XML file=app/manifests/AndroidManifest.xml
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+    <!-- ... rest of the config -->
+    <application
+        android:name=".MyAndroidAnalyticsApplication"
+        <!-- ... rest of the config -->
+    </application>
+</manifest>
+```
+
+To check your setup, build and run your app. Click your button a few times. You should start seeing events in the [activity tab](https://us.posthog.com/events).
+
+<ProductScreenshot
+  imageLight={EventsInPostHogLight} 
+  imageDark={EventsInPostHogDark} 
+  alt="Events captured in PostHog" 
+  classes="rounded"
+/>
+
+## 3. Implement the event capture code
+
+To show how to capture events with PostHog, we capture an event when the submit button on the home page is clicked. To do this, we call [PostHog.capture()](/docs/libraries/android#capturing-events):
+
+```kotlin file=HomeActivity.kt
+// rest of your imports
+import com.posthog.PostHog
+
+class HomeActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        
+        // ...
+
+        submitButton.setOnClickListener {
+            PostHog.capture(event = "home_button_clicked")
+            Toast.makeText(this, "Submitted", Toast.LENGTH_SHORT).show()
+        }
+
+        // ...
+```
+
+Reload your app and click the button on the home page a few times. You should now see the captured event in your [PostHog activity tab](https://us.posthog.com/events).
+
+### Setting event properties
+
+When capturing events, you can optionally include additional information by setting the `properties` argument. This is helpful for breaking down or filtering events when creating [insights](/docs/product-analytics/insights).
+
+As an example, we add the value of the toggle as an event property:
+
+```kotlin file=HomeActivity.kt
+// rest of your imports
+import com.posthog.PostHog
+
+class HomeActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        
+        // ...
+
+        submitButton.setOnClickListener {
+            val isToggleOn = toggleButton.isChecked
+            PostHog.capture(
+                event = "home_button_clicked",
+                properties= mapOf("is_toggled_enabled" to isToggleOn)
+            )
+        }
+
+        // ...
+```
+
+### Identifying users
+
+Linking events to specific users enables you to build a full picture of how they're using your product across different sessions, devices, and platforms. To link events from anonymous to specific users, we call [`PostHog.identify()`](/docs/libraries/android#identify) with a `distinctId` argument. The `distinctId` must be a unique identifier for the user – usually their email or database ID.
+
+To show you an example, update the code for the login button to the following:
+
+```kotlin file=MainActivity.kt
```

Comment:
```suggestion
```kotlin file=MainActivity.kt
// rest of your imports
import com.posthog.PostHog
```